compiler/damlc/tests/src/DA/Test/PositionMapping.hs

```diff
@@ -0,0 +1,88 @@
+{-# LANGUAGE OverloadedStrings #-}
+module DA.Test.PositionMapping (main) where
+
+import Test.Tasty
+import Test.Tasty.HUnit
+
+import Development.IDE.Core.PositionMapping
+import Language.Haskell.LSP.Types
+
+
+main :: IO ()
+main = defaultMain $
+    testGroup "position mapping"
+        [ testGroup "toCurrent"
```

Comment:
I’m planning to extend this test suite with a bunch of property tests but I’ll leave that for a separate PR.